### PR TITLE
chore: Update to multiaddr v3.0.1 - refactor: websocket{s => }-star

### PR DESCRIPTION
**Pull request by mkg20001**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "mafmt",
-  "version": "3.0.0",
+  "version": "3.0.1",
```
Comment (diasdavid) [inline package.json]: @mkg20001 don't update this on PR. This gets updated on release by `aegir`

Comment (mkg20001) [reply package.json]: @diasdavid Updated it

File: src/index.js [hunk 2]
```diff
@@ -44,10 +44,17 @@ const WebRTCStar = or(
...
+  or(
+    or(
+      and(WebSockets, base('p2p-websocket-star'), base('ipfs')),
+      and(WebSocketsSecure, base('p2p-websocket-star'), base('ipfs'))
+    ),
+    or(
+      and(WebSockets, base('p2p-websocket-star')),
+      and(WebSocketsSecure, base('p2p-websocket-star'))
+    )
+  )
```
Comment (diasdavid) [inline src/index.js]: These 4 `or` can be just one `or`

Comment (mkg20001) [reply src/index.js]: There is a bug. When `and(WebSocketsSecure, base('p2p-websocket-star'), base('ipfs'))` comes after `and(WebSockets, base('p2p-websocket-star'))` the tests fail. At the beginning it was like that and I thought that adding 4 ors would fix the bug. Now realized reordering does fix it too.

**Pull request merged**